cinder/volume/drivers/nexenta/iscsi.py

```diff
@@ -118,10 +119,19 @@ def check_for_setup_error(self):
         if not self.nms.volume.object_exists(self.volume):
             raise LookupError(_("Volume %s does not exist in Nexenta SA") %
                               self.volume)
+        if self.folder:
+            folder = '%s/%s' % (self.volume, self.folder)
+            if not self.nms.folder.object_exists(folder):
+                raise LookupError(_("Folder %s does not exist in Nexenta "
```

Comment:
I always thought that 'NexentaStor' is one word? Is it correct to put a space between?

Reply:
you're right